src/extlibs/hrvo/hrvo_agent.h

```diff
@@ -143,8 +143,22 @@ class HRVOAgent : public Agent
 
         // The number of the second velocity obstacle.
         int velocityObstacle2_;
+
+        /**
+         * Returns true if this candidate point intersects the given velocity obstacle,
+         * false otherwise.
+         *
+         * @param vo the velocity obstacle to check against
+         *
+         * @return true if intersecting, false otherwise
+         */
+        bool intersectsVelocityObstacle(const VelocityObstacle &vo) const;
```

Comment:
I feel like "intersects" is a bit ambiguous here. I think it might be a bit more clear if this is moved to `VelocityObstacle` class and it is called something like `containsVector` or `containsVelocity`. What do you think?

Reply:
I agree with that, so I updated it